src/Components/ProgressBar.jsx

```diff
@@ -41,24 +43,35 @@ const ProgressBar = ({ currentSectionName, currentSection, totalSections }) => {
           className="progressBar-current"
           style={{ width: barWidthCurrent }}
         ></div>
-        {stepArr.map((a) => {
-          if (a <= currentSection) {
-            return (
-              <li
-                classname="steps"
-                style={{ marginRight: stepWidth, background: "#4E6C99" }}
-                key={a}
-              >
-                {a}
-              </li>
-            );
-          } else
-            return (
-              <li classname="steps" style={{ marginRight: stepWidth }} key={a}>
-                {a}
-              </li>
-            );
-        })}
+        <li
+          className="steps"
+          style={{ marginRight: stepWidth, background: stepBackgroundActive }}
+          key="step1"
+        >
+          1
```

Comment:
Are there always going to be 3 steps? Can we calculate this dynamically instead of using hard coded values?